link-verifier/verify-links.py

```diff
@@ -14,7 +14,7 @@
 import traceback
 from collections import defaultdict
 
-MARKDOWN_SEARCH_TERM = r'\.md$'
+MARKDOWN_SEARCH_TERM = r"\.md$"
```

Comment:
This still throws a warning about it not being a valid escape sequence? But not sure what it wants.